__tests__/endpoints.test.js

```diff
@@ -41,3 +40,44 @@ describe("GET /api/categories", () => {
       });
   });
 });
+
+describe("GET /api/reviews/:review_id", () => {
+  const expectedProperties = [
+    "review_id",
+    "title",
+    "category",
+    "designer",
+    "owner",
+    "review_body",
+    "review_img_url",
+    "created_at",
+    "votes",
+  ];
+  it("Returns a review object, with following properties, review_id, title, review_body, designer, review_img_url, votes, category field , owner and created_at", () => {
+    return request(app)
+      .get("/api/reviews/4")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body).toBeInstanceOf(Object);
+        expect(body.review_id).toBe(4);
+        const resultKeys = Object.keys(body);
+        expect(resultKeys).toEqual(expectedProperties);
+      });
+  });
+  it("Returns an error message if the id is not present (Out of range)", () => {
+    return request(app)
+      .get("/api/reviews/4444")
+      .expect(400)
```

Comment:
I'd expect this to be a 404 error, as the review id doesn't exist yet, as opposed to an invalid request